packages/earn-controller/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@metamask/earn-controller",
-  "version": "0.14.0",
+  "version": "1.0.0",
```

Comment:
@nickewansmith Would you mind reverting this change? Just so you're aware, updating the changelog ahead of time is fine, but we don't bump the version of a package until we are ready to release that package, and we do that in a separate release PR for visibility. Release PRs get created via `create-release-branch`, and you can read more about that process here: https://github.com/MetaMask/core/blob/main/docs/contributing.md#releasing-changes

``` suggestion
  "version": "0.14.0",
```